examples/miral-shell/spinner/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# -*- Mode: CMake; indent-tabs-mode: nil; tab-width: 2 -*-
+#
+# Copyright © 2014 Canonical Ltd.
```

Comment:
2018

Reply:
Actually, no. This is just a restored file.